simulations/vip-525/bscmainnet.ts

```diff
@@ -0,0 +1,164 @@
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { expectEvents, setMaxStalePeriod, setMaxStalePeriodInChainlinkOracle, setRedstonePrice } from "src/utils";
+import { forking, testVip } from "src/vip-framework";
+
+import vip525, { SUSDE, USDE, XSOLVBTC } from "../../vips/vip-525/bscmainnet";
+import CAPPED_ORACLE_ABI from "./abi/CappedOracle.json";
+import ERC20_ABI from "./abi/ERC20.json";
+import OMNICHAIN_PROPOSAL_SENDER_ABI from "./abi/OmnichainProposalSender.json";
+import RESILIENT_ORACLE_ABI from "./abi/ResilientOracle.json";
+
+const { bscmainnet } = NETWORK_ADDRESSES;
+const REDSTONE_BNB_FEED = "0x8dd2D85C7c28F43F965AE4d9545189C7D022ED0e";
+const SolvBTC = "0x4aae823a6a0b376De6A78e74eCC5b079d38cBCf7";
+export const xSolvBTC_RedStone_Feed = "0x24c8964338Deb5204B096039147B8e8C3AEa42Cc"; // RedStone Price Feed for SolvBTC.BBN_FUNDAMENTAL
+
+const prices = [
+  {
```

Comment:
What about `PT-sUSDE-26JUN2025`?

Reply:
For `PT-sUSDE-26JUN2025` pendle oracle price becomes stale in post VIP simulation. So We have to mock it by configuring it to use redstone price (https://github.com/VenusProtocol/vips/blob/main/vips/vip-517/bscmainnet.ts#L34). If we do that then also we are not validating if the PTsUSDe oracle parameters are set correctly.